tests/gtest/avifincrtest_helpers.cc

```diff
@@ -347,13 +353,30 @@ avifResult DecodeIncrementally(const avifRWData& encoded_avif,
       return AVIF_RESULT_INVALID_ARGUMENT;
     }
     const uint32_t decoded_row_count = avifDecoderDecodedRowCount(decoder);
-    AVIF_CHECKERR(decoded_row_count >= previously_decoded_row_count,
-                  AVIF_RESULT_INVALID_ARGUMENT);
+    if (decoded_row_count < previously_decoded_row_count) {
+      printf("ERROR: decoded row count decreased from %d to %d\n",
+             previously_decoded_row_count, decoded_row_count);
+      return AVIF_RESULT_INVALID_ARGUMENT;
```

Comment:
Consider using `AVIF_CHECKERR(false, AVIF_RESULT_INVALID_ARGUMENT);` to keep the finest break point on `AVIF_BREAK_ON_ERROR`.

Same in other places.